helpers.js

```diff
@@ -292,9 +292,16 @@ const doAnalysis = async (client, config, jsonFiles, contractNames = null, limit
             if (progress) {
                 clearInterval(timer);
                 sleep.msleep(1000); // wait for last setInterval finising
-                bar.tick({
-                    'status': '✗ error'.red
-                });
+                // Check error message from armlet to determine if a timeout occured.
+                if(err.includes("User or default timeout reached after")) {
```

Comment:
I guess the below error message should be hooked also.
https://github.com/ConsenSys/armlet/blob/master/lib/analysisPoller.js#L201-L202

Reply:
thx!